src/location/qt_mapping_engine.cpp

```diff
@@ -227,11 +244,10 @@ QtMappingEngine::QtMappingEngine(const QVariantMap &parameters, QGeoServiceProvi
 }
 
 QGeoMap *QtMappingEngine::createMap() {
-    QGeoMapMapLibre *map = new QGeoMapMapLibre(this);
+    auto map = std::make_unique<QGeoMapMapLibre>(this);
```

Comment:
This feels a bit pointless unless you plan to change the API to return a smart pointer.

Reply:
Either that or an ignore statement. I then decided against any `new` statements in the code.

Reply:
That is a worthy goal but then you should also get rid of raw pointers then eventually I think, because this is just `new` with extra steps. :)

Feel free to resolve.